projects/ngx-d3timeline/src/lib/drag-utils.ts

```diff
@@ -45,41 +15,36 @@ export function getCurrentlyDraggedActivity(
   );
 }
 
-function getDropActivitySeries(
-  bandScale: BandScale,
-  dragEvent: TimelineDragEvent,
-  timeOrientation: Orientation
-) {
-  const invert = getInverseBandScale(bandScale);
-  return dragEvent && timeOrientation === Orientation.Vertical
-    ? invert(dragEvent.x)
-    : invert(dragEvent.y);
-}
-
-function shiftedTimesForDraggingActivity(
+export function shiftUpdatedTimes(
   positionedActivity: PositionedActivity,
-  timeOrientation: Orientation,
-  dragEvent: TimelineDragEvent,
-  timeScale: TimeScale
+  updatedStart: Date,
+  updatedFinish: Date
 ) {
-  const deltaTime = getDeltaTime(timeOrientation, dragEvent);
-
-  const shiftedActivityStart =
-    timeScale(positionedActivity.updatedStart) + deltaTime;
-  const shiftedActivityFinish =
-    timeScale(positionedActivity.updatedFinish) + deltaTime;
+  return (
+    positionedActivity && {
+      ...positionedActivity,
+      updatedStart,
+      updatedFinish
+    }
+  );
+}
 
-  return {
-    updatedStart: timeScale.invert(shiftedActivityStart),
-    updatedFinish: timeScale.invert(shiftedActivityFinish)
-  };
+export function shiftTimeByRangeValue(
+  time: Date,
+  timeScale: TimeScale,
+  rangeValue: number
+): Date {
+  return timeScale.invert(timeScale(time) + rangeValue);
 }
 
-function getDeltaTime(
+export function getDeltaTime(
```

Comment:
`getDeltaTimeForDrag` or `getTimeDeltaForDrag` might read better though we have been putting delta first elsewhere.
or even `getDragTimeDelta`

Reply:
done